.gitignore

```diff
@@ -1,11 +1,6 @@
+/build
+/Resources/doc/_build
+/vendor
 composer.lock
-composer.phar
-.idea
-vendor/
-.DS_Store
-build
 phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
 .php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore